spec/features/view_adoption_app_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+feature 'View a mostly nil Adoption App without crashing', js: true do
+  let!(:admin) {create(:user, :admin)}
+  let!(:adopter_with_null_app) {create(:adopter_with_null_app)}
+
+  scenario "Adoptor barely fills out adoption application" do
+
+    sign_in(admin)
+
+    visit '/adopters'
+    expect(page).to have_content('Adoption Applications')
+    click_link(adopter_with_null_app.name)
+    expect(page).to have_content(adopter_with_null_app.name)
+
+  end
+end
```

Comment:
using Sublime Text 2, ah ah, history lesson found: https://robots.thoughtbot.com/no-newline-at-end-of-file